boards/nxp/ucans32k146/cyphal.px4board

```diff
@@ -0,0 +1,4 @@
+CONFIG_DRIVERS_CYPHAL=y
```

Comment:
Do the Kconfig settings properly disable uavcan / uavcannode for this boardconfig, or do the two modules co-exist nicely?

Reply:
They should however there's was a bug in the SocketCAN stack that only allowed for a single listener.
I've fixed it now in NuttX master 
https://github.com/apache/incubator-nuttx/commit/51a845ce54e38ca6ecc13c2fd7029802cb07076f#diff-f7dcf3e1f8c366b6de10ac9320cbf4ad02b0d82cbe63dbf27d29108777d5c590
Might revisit this driver when the PX4/NuttX gets rebased and we also get the SocketCAN filter changes in.